components/ai_chat/resources/page/components/model_selector/index.tsx

```diff
@@ -101,6 +106,12 @@ export function ModelSelector() {
     [conversationContext.allModels],
   )
 
+  const onClickLearnMore = React.useCallback(() => {
+    const mojomUrl = new Url()
+    mojomUrl.url = NEAR_AI_LEARN_MORE_URL
+    aiChatContext.uiHandler?.openURL(mojomUrl)
```

Comment:
just FYI the Mojom URL isn't a real URL it just wants


```suggestion
    aiChatContext.uiHandler?.openURL({ url: NEAR_AI_LEARN_MORE_URL })
```